server/lib/logger.js

```diff
@@ -1,15 +1,15 @@
 const winston = require('winston');
 
-winston.emitErrs = true;
-
-const logger = new winston.Logger({
+const logger = new winston.createLogger({
+  // handleExceptions: true,
+  format: winston.format.json(),
   transports: [
     new winston.transports.Console({
-      timestamp: true,
+      // timestamp: true,
```

Comment:
If those are now invalid options we can remove the comments?

Reply:
done in [46da9cc](https://github.com/auth0/auth0-authorization-extension/pull/387/commits/46da9cc5fb0dec70c6ea0e1fcb065ba66436c1e6)